Tests/UnitTests/Plugins/Cuda/Seeding/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
-add_executable(SeedfinderCudaTest SeedfinderCudaTest.cpp)
-target_link_libraries(SeedfinderCudaTest PRIVATE ${unittest_extra_libraries} Boost::boost)
+add_executable(SeedFinderCudaTest SeedFinderCudaTest.cpp)
```

Comment:
Quick comment...

You forgot to rename the `SeedfinderCudaTest.cpp` file to `SeedFinderCudaTest.cpp`. :wink: (With CUDA enabled the configuration now fails because of the missing file.)